scss/_toggle-dark.scss

```diff
@@ -0,0 +1,72 @@
+@import "node_modules/bootstrap/scss/functions";
+
+@import "node_modules/bootstrap/scss/variables";
+
+@import 'dark-variables';
+
+@import "node_modules/bootstrap/scss/mixins";
+
+@import 'dark-mixins';
+
+html {
+  font-family: sans-serif; // 2
+  line-height: 1.15; // 3
+  -webkit-text-size-adjust: 100%; // 4
+  -webkit-tap-highlight-color: rgba($black, 0); // 5
+}
+
+body.bootstrap-dark {
+  margin: 0; // 1
+  font-family: $font-family-base;
+  @include font-size($font-size-base);
+  font-weight: $font-weight-base;
+  line-height: $line-height-base;
+  color: $body-color;
+  text-align: left; // 3
+  background-color: $body-bg; // 2
+}
```

Comment:
This is from the _root stylesheet from bootstrap so I don't think a mixin makes sense

Problem is that it's for the body and doing `.bootstrap-dark body` doesn't work because body doesn't have any parent that can have the class

So I made it `body.bootstrap-dark` instead to make it work

Reply:
I'm not sure why you are referring to `.bootstrap-dark body`, but let's leave this code duplication be for now 😉